test-it/karma/utilities/TestHelpers.VireoRunner.js

```diff
@@ -3,10 +3,14 @@
     window.testHelpers = window.testHelpers || {};
 
     var rebootAndLoadVia = function (vireo, viaAbsolutePath) {
-        expect(viaAbsolutePath).toBeNonEmptyString();
+        // Jasmine Matchers library is not always ready in beforeAll so use jasmine core functions
+        expect(typeof viaAbsolutePath).toBe('string');
+        expect(viaAbsolutePath).not.toBe('');
```

Comment:
We want to use the most precise test we can. Some possibilities for this check from least specific to most specific:
isDefined
isString
isNonEmptyString
toMatch(/regexp/)
toEqual
toBe


In places where we can validate the exact string like the statusText is "OK" for HTTP that is as specific as we can get.
The cookies we know a subset match the cookie string.
The paths can be anything besides a non-empty string. We could maybe use (/.via/i) or something but that's unnecessarily strict.